packages/machine/test/integration/bignumber-jest-matcher.ts

```diff
@@ -4,13 +4,21 @@ declare global {
   namespace jest {
     interface Matchers<R> {
       toBeEq(expected: BigNumber): BigNumber;
+      toBeBNAndEq(expected: BigNumber | number): BigNumber;
     }
   }
 }
 
+export function toBeBNAndEq(received: BigNumber | number, argument: BigNumber) {
```

Comment:
I think you could wrap `received` and `argument` in `bigNumberify` in `toBeEq` and get the same effect